static/src/javascripts/projects/commercial/modules/messenger/dfp-origin.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+
+/* eslint-disable */
```

Comment:
I tried this at first, but it doesn't appear to disable the linter. 🤔 I wonder if this is because it is a custom (guardian-frontend) rule. My editor still highlights it with a warning, and the pre-commit hook reports it:
<img width="800" alt="picture 793" src="https://user-images.githubusercontent.com/8607683/27327039-2c3d1b2e-55a5-11e7-8f09-e04e5e87a167.png">